examples/print_match_text.rs

```diff
@@ -0,0 +1,13 @@
+use clap::Parser;
+use tree_sitter_grep::{run_with_callback, Args};
+
+fn main() {
+    let args = Args::parse_from(["tree_sitter_grep", "-q", "(function_item) @f"]);
```

Comment:
Per PR comment, seems non-ideal to force callers to "act as though they're providing a sequence of command-line arguments" in order to "configure" the behavior?